cluster/gce/trusty/configure.sh

```diff
@@ -261,7 +261,7 @@ health_monitoring() {
   # We simply kill the process when there is a failure. Another upstart job will automatically
   # restart the process.
   while [ 1 ]; do
-    if ! timeout 20 docker ps > /dev/null; then
+    if ! timeout 10 docker version > /dev/null; then
```

Comment:
Both `docker version` and `docker info` will return error code 1 when docker daemon is not running. What's the difference?

By the way, I *think* this command was copied from `cluster/saltbase/salt/supervisor/docker-checker.sh` below, which needed to fail on non-zero error code to signal supervisord to restart the script (even when the docker daemon isn't running). In this script, it probably makes sense to distinguish "docker daemon is not running" and "docker daemon fails to respond" as we only need to kill docker in the latter case.